docs/plugins/examples.rst

```diff
@@ -41,20 +41,23 @@ Enable bulk enrollment view in the LMS
 Enable Google Analytics
 =======================
 
+Please be aware that as of May 2023 Google Analytics support has been upgraded from Google Universal Analytics to Google Analytics 4 and you may need to update your configuration as mentioned in the `Open edX docs <https://docs.openedx.org/en/latest/site_ops/how-tos/google-analytics.html>`__.
```

Comment:
Can you move this to either a `.. warning::` or a `.. note::` below the actual code block?

Reply:
@regisb all set